### ISSUE TITLE
Add "broken link checker" to continuous integration tests

### ISSUE THREAD

**Issue by gtback** [STIXProject/stixproject.github.io#200]
I've got a script lying around I've used to do this on an ad-hoc basis. I'll have to clean it up and make it available on Travis.

**Comment by guitarmanvt** [comment]
Broken link on http://stixproject.github.io/documentation/concepts/creating-quality-indicators/ to http://stixproject.github.io/language/profiles/samples/stix_1.2_sample_indicator_sharing_profile_r1.xlsx

**Comment by gtback** [comment]
Yes, the profile XLSX is still on stix.mitre.org not stixproject.github.io, as far as I can tell.

**Comment by gtback** [comment]
In terms of actual link checkers, I would recommend using [this one](https://github.com/wummel/linkchecker). My pretty-basic attempt is [here](https://github.com/gtback/linkchecker), and I've also had success with [this one](https://github.com/alecxe/broken-links-checker).
